python/lsst/summit/utils/astrometry/utils.py

```diff
@@ -263,11 +263,13 @@ def runCharactierizeImage(exp, snr, minPix):
     charConfig.doMeasurePsf = False
     charConfig.doApCorr = False
     charConfig.doDeblend = False
+    charConfig.doMaskStreaks = False
```

Comment:
Probably not an issue for the star trackers, but note that some downstream code may fail if the STREAK mask is not present (I consider that a bug in our code). I believe you are already aware of this, but also note that streak masking is going to move to `detectAndMeasure` on difference images soon, and will not be part of the refactored `calibrateImageTask`.

Reply:
That's great, thanks 👍